pages/database-management/logs.mdx

```diff
@@ -114,6 +114,19 @@ container:
 
 </Steps>
 
+## Hiding sensitive information (Enterprise)
+
+To enhance security, it's crucial to ensure that sensitive information like passwords is not logged. Here's an example of how you can mask passwords in your logs:
+
+Original Log:
+```plaintext
+SET PASSWORD TO 'newpassword' REPLACE 'oldpassword'
+```
+Masked Log:
+```plaintext
+SET PASSWORD TO '****' REPLACE '****'
+```
+In this example, the actual passwords are replaced with asterisks (****) to prevent exposure of sensitive data in the logs. This approach ensures that even if logs are accessed by unauthorized individuals, they won't be able to retrieve the actual passwords.

```

Comment:
Are passwords always replaced with `****`? If they are, rephrase the sentence so that it starts with "The passwords are replaced with..."